resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -171,8 +171,13 @@ private[spark] class Client(
       yarnClient.init(hadoopConf)
       yarnClient.start()
 
-      logInfo("Requesting a new application from cluster with %d NodeManagers"
```

Comment:
How about printing something more interesting instead, like the RM address?

Reply:
nice catch @vanzin i will refactor later.